src/kvstore/RocksEngine.cpp

```diff
@@ -114,6 +125,8 @@ RocksEngine::RocksEngine(GraphSpaceID spaceId,
     db_.reset(db);
     partsNum_ = allParts().size();
     LOG(INFO) << "open rocksdb on " << path;
+
+    backup();
```

Comment:
A little confused, seems like a run for nothing, could give some comments?

Reply:
Long story, I can't remember very clear.
1. If we don't backup here, we write some data, and storage is stopped before we do a valid backup.
2. Mock machine reboot, so rocksdb data in memory is lost. (rocksdb wal is persisted in disk, so we could recover from it in theory)
3. Restart the storage, rocks db will be failed open, log is `While creating a new Db, wal_dir contains existing log file: ...`.

I believe the problem is that if we backup here, it is not nothing here. Something like `CURRENT` and `MANIFEST` will be backuped. In that way, rocksdb will think the wal is legal.

Reply:
I see. It's a very complicated story.